content/04.results.md

```diff
@@ -41,3 +41,25 @@ We observed that including one label function of a mismatched type to distant su
 We found that usually adding random noise did not improve performance (Figure {@fig:auroc_random_label_function_performance} and Supplemental Figure {@fig:aupr_random_label_function_performance}).
 For the CbG edge type we did observe slightly increased performance via AUPR (Supplemental Figure {@fig:aupr_random_label_function_performance}).
 However, performance changes in general were smaller than those observed with mismatched label types.
+
+### Text mined edges can expand a database-derived knowledge graph
+
+![
+Text-mined edges can recall a fair amount of existing edges as well as add a myriad amount of novel edges to Hetionet v1 
+This bar chart shows the number of edges we can successfully recall in green and shows the number of new edges that can be added in blue.  
+Percentages contained in parenthesis represent the fraction of edges we recalled over the total number of edges already contained in Hetionet v1.
+For example, within the compound treats disease edge we can recall 85% of existing edges as well as add 6,088 novel edges.
+](https://raw.githubusercontent.com/danich1/snorkeling/6e929e486537c5d105e393e20984b96910c96024/figures/edge_prediction_experiment/edges_added.png){#fig:hetionet_reconstruction}
+
+Hetionet v1 is a knowledge graph that was constructed from biomedical databases.
+These databases are often populated via some form of manual curation, which can result in missing information.
+One of the goals in our work is to close this gap by populating Hetionet v1 with relationships found within biomedical literature.
+We measured the extent to which this framework can recall preexisting edges as well as add novel edges to Hetionet v1.
+Using a calibrated discriminator model, we scored every candidate sentence within our dataset and grouped candidates based on their mention pair. 
+We took the max score within each candidate group and this score represents the probability of the existence of an edge. 
+We established edges by using a cutoff score that produced an equal error rate between the false positives and false negatives.
```

Comment:
Also in methods. Report the outcome of this procedure here though (i.e., what cutoffs you used)